src/Bridge/Doctrine/Orm/Extension/QueryResultExtensionInterface.php

```diff
@@ -15,12 +15,12 @@
 
 /**
  * Interface of Doctrine ORM query extensions that supports result production
- * for specific cases such as pagination.
+ * for specific cases such as pagination or Query alteration.
  *
  * @author Samuel ROZE <samuel.roze@gmail.com>
  * @author Kévin Dunglas <dunglas@gmail.com>
  */
-interface QueryResultExtensionInterface extends QueryCollectionExtensionInterface
+interface QueryResultExtensionInterface
```

Comment:
Is it necessary to keep this interface? We don't have a parent class to `{Item,Collection}DataProvider`.